fruits.py

```diff
@@ -0,0 +1,246 @@
+import random, pygame, sys, math
```

Comment:
Really your master branch should only have the files that you need to run the program successfully. These redundant or unneeded files should either be on a separate 'dev' branch or not present at all. It took me a while to figure out why you had redundant code :)

Reply:
I saw below that you had a test folder. This file could go there or have it's own or be deleted. Just something to show that it's not part of the main thing beyond just your README